index.js

```diff
@@ -744,6 +744,82 @@ class WebUntis {
         if (response.data.result.code) throw new Error('Server returned error code: ' + response.data.result.code);
         return response.data.result;
     }
+
+    /**
+     * Returns all the Lessons where you were absent including the excused one!
+     * @param {Date} rangeStart
+     * @param {Date} rangeEnd
+     * @param {Integer} execuseStatusId
+     * @param {boolean} [validateSession=true]
+     * @returns {Promise.<Array>}
+     */
+    async getAbsentLesson(validateSession = true, rangeStart, rangeEnd, excuseStatusId = -1) {
+        if (validateSession && !(await this.validateSession())) throw new Error('Current Session is not valid');
+        this._checkAnonymous();
+        const response = await this.axios({
+            method: 'GET',
+            url: `/WebUntis/api/classreg/absences/students`,
+            params: {
+                startDate: this.convertDateToUntis(rangeStart),
+                endDate: this.convertDateToUntis(rangeEnd),
+                studentId: this.sessionInformation.personId,
+                excuseStatusId: excuseStatusId,
+            },
+            headers: {
+                Cookie: this._buildCookies(),
+            },
+        });
+        if (response.data.data == null) throw new Error('Server returned no data!');
+        console.log(response.data.data);
+        return response.data.data;
+    }
+
+    /**
+     * Returns a URL to a unique PDF of all the lessons you were absent
+     * @param {Date} rangeStart
+     * @param {Date} rangeEnd
+     * @param {Integer} execuseStatusId
+     * @param {boolean} lateness
+     * @param {boolean} absences
+     * @param {boolean} [validateSession=true]
+     * @param {boolean} execuseGroup
+     * @returns {String} URL
+     */
+    async getPdfOfAbsentLesson(
+        validateSession = true,
```

Comment:
> I suggest adding parameters with default values at the end of all arguments. 

.

Reply:
Don't forget to update the type definitions ^^

Reply:
Where should i add the type definitions?

Reply:
You forgot to move the parameter https://github.com/SchoolUtils/WebUntis/pull/76/files#diff-7aa4473ede4abd9ec099e87fec67fd57afafaf39e05d493ab4533acc38547eb8R446

Reply:
I see! thx